### ISSUE TITLE
add inline comments for undocumented tests in pull request

### ISSUE THREAD

**Issue by thatkookooguy** [Kibibit/tdd1t#36]
[![Bountysource](https://api.bountysource.com/badge/issue?issue_id=68756225)](https://www.bountysource.com/issues/68756225-add-inline-comments-for-undocumented-tests-in-pull-request?utm_campaign=plugin&utm_content=tracker%2F118903791&utm_medium=issues&utm_source=github)

in phase 1, we simply wrote a passing or failing status on pull requests.

We can show more info to the user by adding inline comments on tests that aren't documented

something like:
```markdown
`<test_name>` was not mentioned in issue\s <issue_numbers>.
Either remove the test or add it to an issue specs to fix the problem.
```

Basically, we need to change `getTestsFromPR` to return an array of `{ testName, row }` to know where to add the comment. **row** should be calculated using the file patches.
